.github/workflows/build-pr.yml

```diff
@@ -48,5 +48,4 @@ jobs:
           GITHUB_USERNAME: x-access-token
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
-          # midlertidig oppsett -DfailIfNoTests=false for å kjøre bygg uten enhetstester. Blir fjernet etter å legge til enhetstester
-          mvn -B --no-transfer-progress verify --settings .m2/maven-settings.xml --file pom.xml -Dtest="no/nav/familie/baks/infotrygd/feed/**" -DfailIfNoTests="false"
\ No newline at end of file
+          mvn -B --no-transfer-progress verify --settings .m2/maven-settings.xml --file pom.xml -Dtest="no/nav/familie/baks/infotrygd/feed/**"
```

Comment:
Trenger man -Dtest? Er en fare for at man tror man kjører tester, men så ble det ikke gjort fordi man la det et feil sted

Reply:
fikset det